var/spack/repos/builtin/packages/exago/package.py

```diff
@@ -46,7 +46,11 @@ class Exago(CMakePackage, CudaPackage, ROCmPackage):
 
     conflicts("~hiop~ipopt", msg="ExaGO needs at least one solver enabled")
 
-    # Dependencides
+    # Profling and optimization flags
+    variant("full_optimizations", default=False, description="Enable/Disable optimizations and release type")
+    #variant("with_profiling", default="none", description="Specify profiling tools", multi=True, values=("none", "hpctoolkit", "tau"))
```

Comment:
Since we aren't implementing profiling here, I think we can stick to just the `full_optimizations` flag for now